cluster-operator/src/test/java/io/strimzi/operator/cluster/model/ListenersValidatorTest.java

```diff
@@ -748,7 +749,7 @@ public void testMinimalConfiguration() {
     @Test
     public void testValidateOauthPlain() {
         KafkaListenerAuthenticationOAuthBuilder authBuilder = new KafkaListenerAuthenticationOAuthBuilder()
-                .withEnableOauthBearer(false);
+                .withEnableOauthBearer(false).withEnablePlain(true).withJwksEndpointUri("http://localhost:8080/jwks").withCheckIssuer(false);
```

Comment:
I think this test test something else from what you are changing. If I read it right, it tests that you cannot disable SASL OAUTHBEARER without adding SASL PLAIN. So IMHO you should keep the original test and add your test as additional one.